### PR TITLE
Adds Configuration

### PR DESCRIPTION
**Pull request by kandhavivekraj**

### CODE REVIEW

**Comment by kandhavivekraj** [comment]
I'll add Readme in free time

File: index.js [hunk 1]
```diff
@@ -5,14 +5,23 @@ const mergeTrees = require('broccoli-merge-trees');
...
-let testAccessibility = process.env.TEST_ACCESSIBILITY;
 let filesTree = [];
 
 module.exports = {
   name: require('./package').name,
 
+  addonOptions: {},
+
+  included(app) {
+   var config = this.project.config();
```
Comment (astronomersiva) [inline index.js]: use `let`

File: index.js [hunk 2]
```diff
@@ -5,14 +5,23 @@ const mergeTrees = require('broccoli-merge-trees');
...
+   var options = config[this.name] || {};
+   this.addonOptions = options;
+
+   this._super.included.apply(this, app);
```
Comment (astronomersiva) [inline index.js]: have you tested this with engines once?

File: tests/dummy/config/accessibility.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+module.exports = {
+  testAccessibility: true,
```
Comment (astronomersiva) [inline tests/dummy/config/accessibility.js]: can this be just `enabled`?

File: tests/dummy/config/accessibility.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+  axeConfig: {
```
Comment (astronomersiva) [inline tests/dummy/config/accessibility.js]: this can be just `axe`

**Pull request merged**